tests/cases/conformance/override/override13.ts

```diff
@@ -0,0 +1,17 @@
+// @noImplicitOverride: true
+// @target: esnext
+
+class Foo {
+    property = 1
+    static staticProperty = 2
+}
+
+class SubFoo extends Foo {
+    property = 42;
+    staticProperty = 42;
+}
+
+class StaticSubFoo extends Foo {
+    static property = 42;
+    static staticProperty = 42;
+}
```

Comment:
Can we add a test that ensures this works more than one heritage clause removed?

```ts
class Base {
    static foo = 0;
}

class Intermediate extends Base {}

class Derived extends Intermediate {
    static foo = 1;
}
```